torch/_inductor/kernel/flex_attention.py

```diff
@@ -855,18 +855,28 @@ def flex_attention(
     for BLOCK_M, BLOCK_N, num_warps, num_stages in configs:
         if SPARSE_KV_BLOCK_SIZE % BLOCK_N != 0 or SPARSE_Q_BLOCK_SIZE % BLOCK_M != 0:
             continue
-        # Work around https://github.com/pytorch/pytorch/issues/129625
-        if num_stages == 2:
-            continue
 
         cur_kernel_options = original_kernel_options.copy()
         # Performance tuning
+        # Triton parameters
+        cur_kernel_options.setdefault("fwd_num_stages", num_stages)
+        cur_kernel_options.setdefault("fwd_num_warps", num_warps)
+        # Work around https://github.com/pytorch/pytorch/issues/129625
```

Comment:
can you try removing this, the PR mentioned in In the Triton issue has since been upstreamed to pie torches in Triton.